runtime/caml/backtrace.h

```diff
@@ -69,12 +70,17 @@
  *
  * Its maximum size is determined by [BACKTRACE_BUFFER_SIZE] from
  * [backtrace_prim.h], but this shouldn't affect users.
- *
- * [Caml_state->backtrace_last_exn] stores the last exception value that was
+ */
+#define caml_backtrace_buffer (Caml_state_field(backtrace_buffer))
+#define caml_backtrace_pos (Caml_state_field(backtrace_pos))
+
+/* [Caml_state->backtrace_last_exn] stores the last exception value that was
  * raised, iff [Caml_state->backtrace_active != 0]. It is tested for equality
  * to determine whether a raise is a re-raise of the same exception.
- *
- * FIXME: this shouldn't matter anymore. Since OCaml 4.02, non-parameterized
+ */
+#define caml_backtrace_last_exn (Caml_state_field(backtrace_last_exn))
+
+/* FIXME: this shouldn't matter anymore. Since OCaml 4.02, non-parameterized
```

Comment:
Answering a question of @gadmm , those variables are not packed together because the comments in the header details each of them, so it seemed natural to synchronize the documentation with the code. Nevertheless, since there are no other variables defined in-between, it should be easy to add export-control macros around if needed.